bumble/hci.py

```diff
@@ -732,19 +727,13 @@ def phy_list_to_bits(phys: Optional[Iterable[int]]) -> int:
 
 HCI_LE_PHY_BIT_NAMES = ['LE_1M_PHY', 'LE_2M_PHY', 'LE_CODED_PHY']
 
-HCI_LE_PHY_TYPE_TO_BIT = {
-    HCI_LE_1M_PHY:    HCI_LE_1M_PHY_BIT,
-    HCI_LE_2M_PHY:    HCI_LE_2M_PHY_BIT,
-    HCI_LE_CODED_PHY: HCI_LE_CODED_PHY_BIT
+HCI_LE_PHY_TYPE_TO_BIT: dict[int, int] = {
```

Comment:
should this be `dict[Phy, int]`?

Reply:
Done. This will requires some device API changes as well.